DESIGN_GUIDELINES.md

```diff
@@ -119,6 +119,21 @@ Existing Inconsistencies would not be published, that’s for our internal use 
 | apiGatewayRole	| iam.Role	||
 
 
+## WAF WebACL
+**Required Attributes on Props** 
+
+| Name    | Type     | Notes    |
+| --- | --- | --- |
+| existingWebaclObj? | waf.CfnWebACL	||
+| webaclProps?	| waf.CfnWebACLProps	||
```

Comment:
Are these supposed to be `aws_wafv2.CfnWebACLProps` & `aws_wafv2.CfnWebACL` ?

Reply:
Yes they are from wafv2 but were imported as waf, will update for clarity